e2e/interchaintestv8/ibc_eureka_test.go

```diff
@@ -494,39 +490,39 @@ func (s *IbcEurekaTestSuite) TestICS20Transfer() {
 			s.Require().NoError(types.GenerateAndSaveFixture("acknowledgePacket.json", s.contractAddresses.Erc20, "ackPacket", msg, sendPacket))
 		}
 
-		s.Require().True(s.Run("Verify balances", func() {
-			// Check the balance of the SdkICS20Transfer.sol contract
-			ics20Bal, err := s.erc20Contract.BalanceOf(nil, ics20Address)
+		s.Require().True(s.Run("Verify balances on Ethereum", func() {
+			// User balance on Ethereum
+			userBalance, err := s.erc20Contract.BalanceOf(nil, ethereumUserAddress)
 			s.Require().NoError(err)
-			s.Require().Equal(testvalues.ERC20TransferAmount, ics20Bal.Int64())
+			s.Require().Equal(testvalues.InitialBalance-testvalues.TransferAmount, userBalance.Int64())
 
-			// Check the balance of the sender
-			userBalance, err := s.erc20Contract.BalanceOf(nil, userAddress)
+			// ICS20 contract balance on Ethereum
+			ics20Bal, err := s.erc20Contract.BalanceOf(nil, ics20Address)
 			s.Require().NoError(err)
-			s.Require().Equal(testvalues.StartingERC20TokenAmount-testvalues.ERC20TransferAmount, userBalance.Int64())
+			s.Require().Equal(testvalues.TransferAmount, ics20Bal.Int64())
 		}))
 	}))
 
 	var returnPacket channeltypes.Packet
-	s.Require().True(s.Run("Transfer back", func() {
+	s.Require().True(s.Run("Send transfer on Cosmos chain", func() {
```

Comment:
I wonder if it would be clearer to read the flow if it has some indication that this is a transfer back of the same tokens?

Perhaps:
```suggestion
	s.Require().True(s.Run("Transfer tokens back from Cosmos chain", func() {
```

or something like that?

Reply:
Yes, I agree!